src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -92,40 +219,78 @@ public void CheckProofOfStake(ContextInformation context, ChainedBlock pindexPre
             this.logger.LogTrace("(-)[OK]");
         }
 
-        private bool IsConfirmedInNPrevBlocks(UnspentOutputs utxoSet, ChainedBlock pindexFrom, long maxDepth)
+        /// <summary>
+        /// Stake Modifier (hash modifier of proof-of-stake):
```

Comment:
the purpose of the stake modifier should rather be in the big picture comment, not here
this method does not define the purpose, so its summary should describe what the method does